com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -1781,6 +1812,10 @@ static public EOSSingleton Instance
         {
             get
             {
+                // The EOSSingleton object requires EOSManager to be around and alive in order to run its updates
+                // So when the Instance is asked for, make sure there's an EOSManager
+                _ = ManagerInstance;
```

Comment:
It is not clear to me how this line of code accomplishes what the comment above it describes.

Reply:
It accesses the `get` part of the static property, which will resolve it and create the ManagerInstance if it needs to.